examples/complete/stacks/mixins/region/us-east-1.yaml

```diff
@@ -4,7 +4,7 @@ vars:
 
 components:
   terraform:
-    vpc:
+    infra/vpc:
```

Comment:
How come we use this `infra/` prefix here?

I ask this because I like to link to the `examples/complete` directory in this repo to give people an example of what the yaml stacks and root terraform components look like in a common client engagement and the `infra/` prefix for the `vpc` component is unique only to this repo.

Can we omit the `infra/` dir path here?

Reply:
Maybe we can use `eks/cluster` for directory specific components then